src/main/java/com/ldtteam/structurize/util/TickedWorldOperation.java

```diff
@@ -165,7 +169,47 @@ public boolean apply(final ServerWorld world)
 
         if (operation == OperationType.PLACE_STRUCTURE)
         {
-            currentPos = wrapper.placeStructure(world, storage, currentPos);
+            StructurePhasePlacementResult result;
+            switch (structurePhase)
+            {
```

Comment:
not a fan of these hardcoded int-phases :D

Reply:
Those are only for this specific use case. Other handlers will most likely have their own phases.

Reply:
still at least an enum? xD

Reply:
An enum, for one time local use? Nah